### DISCUSSION TITLE
Have PHP Pecl installed by default

### DISCUSSION THREAD

**Discussion by zaherg** [community/community#25881] [Actions]
<p>Hey,</p>
<p>It would be nice to have PHP Pecl installed by default since it is a painful process to install it on every single job, just to install an extension required by our application.</p>
<p>example <code>ext-intl</code> is not installed by default, and running</p>
<pre><code>apt-get install php73-intl
</code></pre>
<p>will fail because of a fast-cgi error.</p>
<p>Thanks</p>

**Comment by ethomson** [comment]
<p>Thanks for the feedback, we’ll take a look.</p>

**Comment by ADmad** [comment]
<p>Are the build scripts for generating the VMs used by Github Actions publicly available as there are for Azure pipelines here <a href="https://github.com/microsoft/azure-pipelines-image-generation?" rel="nofollow noopener">https://github.com/microsoft/azure-pipelines-image-generation? </a> That would allow the community to easily submit updates/sugestions from improving the VMs.</p>

**Comment by ethomson** [comment, marked as answer]
<p>Not yet, but they will be soon.  You can raise issues in <a href="https://github.com/actions/virtual-environments" rel="nofollow noopener">https://github.com/actions/virtual-environments</a> and we’ll be uploading the code there soon and will accept pull requests.</p>
